### PR TITLE
Fix event type navigation

### PR DESCRIPTION
**Pull request by afzalsayed96**
This PR fixes the following issues:
1. Make entire dropdown item width clickable
Screenshot 2022-09-18 at 2 51 11 AM

2. ~Hitting browser back button on `/v2/event-types/[event]?tabName` button should take you back to `/event-types`~ page

https://user-images.githubusercontent.com/14029371/190878899-5ddcdd0d-de0c-4d3d-ba61-6c14831163ce.mov

3. #4501

### CODE REVIEW

**Comment by PeerRich** [comment]
just tested your PR:

it would be nice if:

- the border radius stays applied (maybe overflow-hidden?)
- the bottom separator between embed and delete is being overwritten by black border

https://user-images.githubusercontent.com/8019099/190900398-c45a4af8-89e1-4036-b366-35aec5ca478a.mp4

**Comment by afzalsayed96** [comment]
@PeerRich fixed

File: apps/web/components/v2/eventtype/EventTypeSingleLayout.tsx [hunk 1]
```diff
@@ -141,7 +141,7 @@ function EventTypeSingleLayout({
   useEffect(() => {
     // Default to the first in the list
     if (!router.query.tabName) {
-      router.push({
+      router.replace({
```
Comment (PeerRich) [inline apps/web/components/v2/eventtype/EventTypeSingleLayout.tsx:144]: why does the /v2/ redirect appear in the first place, anyway? @zomars https://github.com/calcom/cal.com/issues/4579

Comment (PeerRich) [reply apps/web/components/v2/eventtype/EventTypeSingleLayout.tsx:144]: i'll need to wait for @zomars for this to be merged

Comment (zomars) [reply apps/web/components/v2/eventtype/EventTypeSingleLayout.tsx:144]: This code got deprecated in 588748d

**Review by emrysal — APPROVED** [review]

**Comment by emrysal** [comment]
Much appreciated @afzalsayed96 - checked your code out and the behaviour was great; thanks for the following-up on your original PR when requested.

**Pull request merged**